stackstac/rio_reader.py

```diff
@@ -64,7 +63,10 @@ def _curthread():
 # See `ThreadLocalRioDataset` for more.
 # https://github.com/pangeo-data/pangeo-example-notebooks/issues/21#issuecomment-432457955
 # https://gdal.org/drivers/raster/vrt.html#multi-threading-issues
+
 MULTITHREADED_DRIVER_ALLOWLIST = {"GTiff"}
+DEFAULT_SCALE = 1
+DEFAULT_OFFSET = 0
```

Comment:
nit: I'd actually rather not have these be pulled out into variables, because they're not something that could ever change (DEFAULT_SCALE is never going to be 2, for instance), and the fact that they're 0 and 1 are somewhat significant to the code (`if scale != DEFAULT_SCALE: result *= scale` is strange unless you know `DEFAULT_SCALE` is 1) . To me, it's easier to just read/write `0` instead of `DEFAULT_OFFSET`.

Reply:
Sounds good